src/test/java/TrainingTimeManagerTest.java

```diff
@@ -0,0 +1,103 @@
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class TrainingTimeManagerTest {
+
+    /* calculateHoursBetweenTwoDateTimes method: */
+    @Test
+    public void calculateHoursBetweenTwoDateTimes_SameDayTimeDifference() {
+        assertEquals(7, TrainingTimeManager.calculateHoursBetweenTwoDateTimes(
```

Comment:
I understand that this was not present in the course, but I would suggest you to use asserts from the `assertj` library, as they are more flexible and it is very likely that you would face this library in a real project. So it is better to get familiar with it now.